sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/RxDocumentClientImpl.java

```diff
@@ -4488,4 +4490,33 @@ private static List<FeedRange> toFeedRanges(
     private static FeedRange toFeedRange(PartitionKeyRange pkRange) {
         return new FeedRangeEpkImpl(pkRange.toRange());
     }
+    /**
+     * This class generates fast UUID instances using a pseudo random number generator.
+     * It should be used when speed is more important than security.
+     *
+     *
+     * */
+    class UUIDGenerator {
```

Comment:
We have a class with the same name in uuid package, we should use a different name here to avoid confusion.